src/locutus/model/user_input.py

```diff
@@ -148,12 +151,6 @@ def create_or_replace_user_input(self, resource_type, collection_type, id, code,
             if type not in existing_data:
                 existing_data[type] = user_input_instance.return_format()
 
-            # Get user_id to identify existing data for the user.
-            try:
-                user_id = SessionManager.create_user_id(editor=editor)
-            except ValueError as e:
-                print(f"An error occurred while getting the user_id. {e}")
-
```

Comment:
editor is already set above. Just change  ln 172 to use editor instead of user_id